debian/ORIG_URL

```diff
@@ -0,0 +1 @@
+https://amazon-ecr-credential-helper-releases.s3.us-east-2.amazonaws.com/0.3.1/release-novendor.tar.gz
```

Comment:
Is there any reason not to grab the main release tarball and strip out the vendored components in the docker-deb script? Or, assuming the "upstream" releases are tagged in the repo, use `gbp export-orig` to construct the .orig.tar.xz file directly from the repo?

Reply:
I think there are multiple valid ways to get a tarball.  The way I did this before was to use a make target in the upstream repo that generated a tarball that excluded the vendor folder and included a generated `GITCOMMIT_SHA` file and then used that.  This approach basically does the same thing, except that now the vendor-less tarball is actually published in a durable location and doesn't need to be regenerated.

I didn't really want to use the docker-deb script to modify the tarball itself as I didn't want the docker-deb script to be the only way to build this package.

As for `gbp export-orig`, I'm not very familiar with that.  Currently installing it in a container to go look at it more closely, but I don't know if it provides a mechanism to exclude the vendor folder or would give me a tarball that has the `GITCOMMIT_SHA` file.

Reply:
You don't need `gbp export-orig` to export the final file; you can post-process it as needed.

Reply:
Is there a preference for doing that over using an already-processed file that's published for this purpose?

Reply:
It doesn't really matter. If the stripped version is useful beyond just the .deb package, then it's worth publishing upstream. If not, then it's pretty much a wash. The Debian archive will become the durable storage for the .orig file once it's uploaded, so so nothing is lost by not hosting it in s3.

Reply:
Got it.  The stripped version is already there and there's automation around producing it at the same time as the other release artifacts.  I also expect that other packagers who also don't want the vendor folder (for example, if this were to be packaged in Fedora) would find it useful.

At some point I plan to transition this project to using Go modules and away from vendoring entirely, so the difference between the full and the stripped tarball will go away.